metrics/sqlite_query.py

```diff
@@ -0,0 +1,56 @@
+import sys
+import logging
+from sqlalchemy import create_engine, or_
+from sqlalchemy.orm import sessionmaker
+from db.models import Base, Issue, IssueState
+
+
+class SQLiteQuery:
+
+    def __init__(self, database) -> None:
+        self.logger = logging.getLogger(self.__module__)
+        self.logger.setLevel(logging.DEBUG)
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
+        formatter = logging.Formatter(
+            '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
+        engine = create_engine(f"sqlite:///{database}")
+        Base.metadata.create_all(engine, Base.metadata.tables.values(), checkfirst=True)
+        session_maker = sessionmaker(bind=engine)
+        self.session = session_maker()
+
+    def _check_session(self):
+        if self.session is None:
+            raise Exception("The session has to be inizializate")
+
+    def status_snapshot(self, date, **filter):
```

Comment:
```suggestion
    def status_snapshot(self, **filter, date = None):
```
otherwise your code below won't work . skipping date will throw "need 3 params but only 2 given " . Also you need to put required params before optional

Reply:
```
************* Module sqlite_query
metrics/sqlite_query.py:24:41: E0001: Parsing failed: 'invalid syntax (<unknown>, line 24)' (syntax-error)
```
And pylance says
```
Parameter cannot follow "**" parameter - Pylance
```
Anyway with no date is the same than get all the issues, so I created a new method, issues to get them. Therefore date now is mandatory can cannot be None